kedro-org/kedro-website
# Feature/blog article page

## Description

Resolves #80.

## Development notes

This creates and styles the blog post page from the ground up.

It's quite a large PR. Sorry about that. Thankfully nothing in here is too complicated, code wise, so it should be easy enough to review, even though there are a lot of changes. Happy to pair on the review if it's easier.

## QA notes

[Take a look at the deployed Netlify URL](https://deploy-preview-81--kedro-website.netlify.app/blog/what-is-kedro-viz-2) and check it against the designs.

_Note_: instead of the design's **Related posts** at the bottom of the page, I've added **All blog posts** instead, since we don't have enough posts or content for a related-posts area.

I'm still missing a some small things:

- [x] Back to blog home icon (need from @Mackay031)
- [ ] Design guidance on what should happen when a user clicks **Copy link** (need from @Mackay031)
- [ ] Better UX behavior with the sticky nav. There was no design spec for that, so for now, it's a minimal implementation. (need from @Mackay031 if we want to change from how it is now)

## Checklist
